src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py

```diff
@@ -8,18 +8,14 @@
 from tempfile import gettempdir
 from uuid import uuid4
 
-path = '/usr/share/archivematica/dashboard'
-if path not in sys.path:
-    sys.path.append(path)
+import django
+sys.path.append('/usr/share/archivematica/dashboard')
```

Comment:
That is a great idea, looking into it.  We use threads not subprocesses currently, so I dunno how different it'll be...